addons/event_booth_sale/i18n/event_booth_sale.pot

```diff
@@ -334,6 +334,14 @@ msgid ""
 "%(booth_names)s"
 msgstr ""
 
+#. module: event_booth_sale
```

Comment:
there are still changes in this pot file :eyes: :slightly_smiling_face:

Reply:
yes, both of those are used in the ValidationError. I'm confused, something changed in the way we handle translations ?

Reply:
Well we don't need to update the translation files when targeting master, it will be automatically added later on :slightly_smiling_face: